.gitignore

```diff
@@ -18,3 +18,5 @@ target
 
 ## Hippo content
 /repository-data/application/src/main/resources/hcm-content/content/documents/publicationsystem/publications/*
+
+repository-data/application/src/main/resources/hcm-config/hst/hosts/*
```

Comment:
I had to add my local IP to the hosts, so that I can test my local version of the app from Windows VMs via Virtualbox. This line in the ignore file was added to avoid accidental committing of my hosts file updates.

Reply:
surly you wanted to modify the one in "development" module, which is fine to commit. I'm sure more people would use it.